samples/drivers/biometrics/fingerprint/README.rst

```diff
@@ -0,0 +1,90 @@
+.. _fingerprint_sensor_sample:
+
+Fingerprint Sensor Sample
+##########################
+
+Overview
+********
```

Comment:
this needs to use `.. zephyr:code-sample::` (template here https://github.com/zephyrproject-rtos/zephyr/blob/main/doc/templates/sample.tmpl or see other code samples for reference)